test/mock-settlement.js

```diff
@@ -1,19 +1,21 @@
+const { GBP } = require('../app/constants/currencies')
+const { AP } = require('../app/constants/ledgers')
+
```

Comment:
Any reason or rule for white-space?

Reply:
Separate values coming from `constants` (above) versus those from `mock-components` (below)

Reply:
no further action taken